plutus-ledger-api/src/PlutusLedgerApi/V1/Value.hs

```diff
@@ -262,6 +261,15 @@ symbols (Value mp) = Map.keys mp
 singleton :: CurrencySymbol -> TokenName -> Integer -> Value
 singleton c tn i = Value (Map.singleton c (Map.singleton tn i))
 
+lovelaceValue :: Lovelace -> Value
```

Comment:
why does `lovelaceValue` not need to be inlinable?

Reply:
Good catch. I overlooked that.